test/fixtures/html/basic-include.html

```diff
@@ -6,5 +6,9 @@
   </head>
   <body>
     <!--=include test.html -->
+    <!-- = include test.html -->
```

Comment:
Hi! For what it's worth (I added the HTML syntax): the readme explains a syntax without spaces around the `=` sign, and I think it would be best to not give the option to users to include spaces at that place at all. Better to have one, dictated syntax over all file types, without margin for errors. (I might type a normal comment starting with `<!-- = ... ` without the intent to use an include.)

Just my thoughts. :)